internal/app/pactproxy/proxy.go

```diff
@@ -12,27 +13,47 @@ import (
 	"time"
 
 	"github.com/form3tech-oss/pact-proxy/internal/app/httpresponse"
+	"github.com/sethvargo/go-envconfig"
 	log "github.com/sirupsen/logrus"
 )
 
-const (
-	defaultDelay    = 500 * time.Millisecond
-	defaultDuration = 15 * time.Second
-)
+type Config struct {
+	ServerAddress url.URL       `env:"SERVER_ADDRESS"`            // Address to listen on
+	Proxies       []url.URL     `env:"PROXIES,delimiter=;"`       // List of URL to serve pact-proxy on, e.g. http://localhost:8080;http://localhost:8081
+	AdminPort     int           `env:"ADMIN_PORT,default=8080"`   // Port for Admin server
+	WaitDelay     time.Duration `env:"WAIT_DELAY,default=500ms"`  // Default Delay for WaitForInteractions endpoint
+	WaitDuration  time.Duration `env:"WAIT_DURATION,default=15s"` // Default Duration for WaitForInteractions endpoint
+	Target        url.URL       // Do not load Target from env, we set this for each value from Proxies
+}
+
+func (c *Config) SetDefaults() {
+	defaultConfig := Config{}
+	// Can be changed (on next release) to: envconfig.ExtractDefaults(context.Background(), &defaultConfig)
```

Comment:
should we carry this information in a issue ticket instead of a comment?

Reply:
Yes, but I've decided to remove SetDefaults in favor of a simpler approach